src/runtime/entries/cloudflare-pages.ts

```diff
@@ -44,8 +44,7 @@ export async function onRequest(ctx: CFRequestContext) {
     host: url.hostname,
     protocol: url.protocol,
     body,
-    // TODO: Allow passing custom context
-    // cf: ctx,
+    context: ctx,
```

Comment:
**Note:** this needs to be called `context` and not `cf` (as suggested in the comment) because unenv takes this specific context field and puts it in the `req.__unenv__`: https://github.com/unjs/unenv/blob/4ed234cf8fd88264852739d14b40bebf7ce31afe/src/runtime/fetch/call.ts#L51
(which is where then users can have access to it)

Reply:
Sure but it can be nested like here: https://github.com/unjs/nitro/blob/7bcfc362a77764aed566b9f266ae702ccd0dceed/src/runtime/entries/cloudflare.ts#L36

Reply:
It is intentionally verbose because we are accessing a platform-api. `cf` is already added for normal worker target. we need to add it it for pages too (i suppose interfaces are same)

Reply:
ah by the way, regarding the `cf` we I don't think we need to manually pass it since it will accessible from the `ctx.request` anyways

eg.
![Screenshot at 2023-02-27 15-15-19](https://user-images.githubusercontent.com/61631103/221603189-af2f187e-8435-4ff4-8267-93452193ca4a.png)

so it looks like `context: ctx,`  should actually be enough, what do you think?

or alternatively we can and choose what fields from the CfRequestContext pass along, in that case we could construct the `context` ourselves, but I am not sure if that's needed/beneficial

Reply:
`event.context.cf` can have all the keys and context passed by cloudflare. that simple :)

Reply:
mh.... ok, but if we do that then if someone uses the worker adaptor then the `event.context.cf` will contain only the cf object whilst with the pages adaptor `event.context.cf` will contain the `CfRequestContext`, so this might lead to confusion?

if you are ok with this misalignment of the two I am as well, I'd also want to update the worker adaptor anyways in a followup PR to make it use the ESM syntax, so I could re-align them there (but I wanted to get this pages change out of the way asap so that people can start using Nuxt on Cloudflare pages, the worker adaptor refactoring is not that pressing)